common/scripted_effects/hfe_effect_population.txt

```diff
@@ -1,117 +1,131 @@
 #
 
 hfe_effect_population_country_set_robot_pop_identity = {
-	# if = {
-	# 	limit = {
-	# 		has_trait = trait_machine_unit
-	# 	}
-	# 	every_owned_pop_species = {
-	# 		limit = {
-	# 			has_trait = trait_machine_unit
-	# 			not = {
-	# 				is_same_species = prev.species
-	# 			}
-	# 		}
-	# 		root = {
-	# 			modify_species = {
-	# 				base = none
-	# 				species = prev
-	# 				add_trait = hfe_species_trait_test
-	# 			}
-	# 		}
-	# 		set_species_identity = prev.species
-	# 		root = {
-	# 			modify_species = {
-	# 				species = prev
-	# 				remove_trait = hfe_species_trait_test
-	# 			}
-	# 		}
-	# 	}
-	# }
-	# else_if = {
-	# 	limit = {
-	# 		has_trait = trait_mechanical
-	# 	}
-	# 	every_owned_pop_species = {
-	# 		limit = {
-	# 			has_trait = trait_mechanical
-	# 			not = {
-	# 				is_same_species = prev.species
-	# 			}
-	# 		}
-	# 		root = {
-	# 			modify_species = {
-	# 				base = none
-	# 				species = prev
-	# 				add_trait = hfe_species_trait_test
-	# 			}
-	# 		}
-	# 		set_species_identity = prev.species
-	# 		root = {
-	# 			modify_species = {
-	# 				species = prev
-	# 				remove_trait = hfe_species_trait_test
-	# 			}
-	# 		}
-	# 	}
-	# }
-	# else = {
-	# 	random_owned_pop = {
-	# 		limit = {
-	# 			has_trait = trait_mechanical
-	# 		}
-	# 		save_event_target_as = hfe_robot_pop_species
-	# 	}
-	# 	every_owned_pop_species = {
-	# 		limit = {
-	# 			has_trait = trait_mechanical
-	# 			exists = event_target:hfe_robot_pop_species
-	# 			not = {
-	# 				is_same_species = event_target:hfe_robot_pop_species
-	# 			}
-	# 		}
-	# 		root = {
-	# 			modify_species = {
-	# 				base = none
-	# 				species = prev
-	# 				add_trait = hfe_species_trait_test
-	# 			}
-	# 		}
-	# 		set_species_identity = prev.species
-	# 		root = {
-	# 			modify_species = {
-	# 				species = prev
-	# 				remove_trait = hfe_species_trait_test
-	# 			}
-	# 		}
-	# 	}
-	# }
+	if = {
```

Comment:
To what are you referring?

Reply:
Do you mean the integration of Robot Pops from Machine Units to Mechanical Units and vice-versa?

Reply:
I commented it out for a reason.

Reply:
This was discussed in freaking April. Why are you raising the dead?

The conversion of Robots to Machines and vice-versa is by far one of the best features of the Mod. If you promise me that you'll add it back, I'll put the pound signs back.

Reply:
Yes, I will add it back.

Reply:
Okay, done. I trust your word.

BTW, in the future, can you make it so that it checks ALL robots instead of just switching them on conquered planets? For example, when Pirates capture a robot with their bombardment, it should change them there, too.